utility/map_str_enhancer.go

```diff
@@ -51,3 +106,14 @@ func MillisAsMicros(ms float64) common.MapStr {
 	m["us"] = int(ms * 1000)
 	return m
 }
+
+func AddStrWithDefault(m common.MapStr, key string, val *string, defaultVal string) {
```

Comment:
this is not used anywhere?

Reply:
This is not a new function, I only moved it, but you're right, so I'll remove it.